doc/rofi.1.markdown

```diff
@@ -571,30 +564,30 @@ Allow multiple lines to be selected. Adds a small selection indicator to the lef
 
 `-sync`
 
-Force rofi mode to first read all data from stdin before showing selection window. This is how original dmenu behaviour.
+Force rofi mode to first read all data from stdin before showing the selection window. This is original dmenu behavior.
 
-Note: the default asynchronous mode will also be automatically disabled if used with conflicting options
-such as `-dump`, `-only-match` or `-auto-select`
+Note: the default asynchronous mode will also be automatically disabled if used with conflicting options,
+such as `-dump`, `-only-match` or `-auto-select`.
 
 `-async-pre-read` *number*
 
-Reads the first 25 entries blocking. Then switches to async mode. This makes it feel more 'snappy'.
+Reads the first 25 entries blocking, then switches to async mode. This makes it feel more 'snappy'.
 
 *default*: 25
 
 ### Message dialog
 
 `-e` *message*
 
-Popup a message dialog (used internally for showing errors) with *message*.
+Pops up a message dialog (used internally for showing errors) with *message*.
 Message can be multi-line.
 
 ### Other
 
 `-pid` *path*
 
-Make **rofi** create a pid file and check this on startup. Avoiding multiple copies running
-simultaneously. This is useful when running **rofi** from a keybinding daemon.
+Make **rofi** create a pid file and check this on startup. Avoid multiple copies running
```

Comment:
Should it be "avoid" as in imperative? Or something else?
If it's advice, then you could use an additional sentence explaining why, for example:
>Avoid running multiple copies, because...